### PR TITLE
Feature/ui biopsy question changes

### PR DESCRIPTION
**Pull request by achen2401**
address this:
https://www.pivotaltracker.com/story/show/141833645
- modify current biopsy question as appeared in initial queries and profile page to include new unknown option and a new date field for the biopsy date

minor fixes to clinics selector as raised in this issue:https://www.pivotaltracker.com/story/show/141769661

### CODE REVIEW

File: portal/static/js/main.js [hunk 1]
```diff
@@ -249,21 +249,46 @@ var CONSENT_ENUM = {
...
-                }
+                if(!$radios.is(':checked')) {
+                    if (status == "unknown") $radios.filter('[value="unknown"]').prop('checked', true);
+                    else $radios.filter('[value='+clinicalValue+']').prop('checked', true);
+                    if (clinicalItem == "biopsy") {
+                        if (clinicalValue == "true") {
+                            var issuedDate = val.content.issued;
+                            var d = issuedDate.substring(0, 10).split("-");
+                            //in DD/MM/YYYY format
+                            $("#biopsyDate").val(d[2] + "/" + d[1] + "/" + d[0]);
```
Comment (ivan-c) [inline portal/static/js/main.js]: I worry this is a little fragile, does it make sense to use `.toLocaleString()` everywhere, or do you think that's overkill?

File: portal/templates/initial_queries_macros.html [hunk 2]
```diff
@@ -112,16 +112,26 @@ <h4 class="tnth-headline">{{ _("Terms of Use") }}</h4>
...
           <div class="form-group">
-            <div class="radio">
+            <div class="radio biopsy-option">
               <label>
                 <input type="radio" class="init-queries-field" name="biopsy" id="biopsy_yes" value="true"> {{ _("Yes") }}
               </label>
             </div>
-            <div class="radio">
+            <div id="biopsyDateContainer" class="tnth-hide indent form-group">
+            	<label for="biopsyDate">{{_("Biopsy Date: ")}}</label>
```
Comment (ivan-c) [inline portal/templates/initial_queries_macros.html]: You might want to move the colon outside the translation call (ie `{{_("Biopsy Date")}}:`), that way you can reuse that translation other places in the UI without having to add translations with and without a colon.

**Comment by achen2401** [comment]
@ivan-c 
Ivan, I am waiting for Adam to make modifications of API to allow me to remove/replacing existing observation(s).  So need to hold off merging these changes for now.   But I did make changes according to your comments.

**Comment by achen2401** [comment]
Ivan, this PR should be ready now.  I tested my changes in Chrome, Safari and Firefox and IE.  Please review at your convenience.  Thank you!

**Pull request merged**